tests/jepsen/run.sh

```diff
@@ -298,34 +298,6 @@ case $1 in
         RUN_JEPSEN "test $CONTROL_LEIN_RUN_ARGS"
         end_time="$(docker exec jepsen-control bash -c 'date -u +"%Y%m%dT%H%M%S"').000Z"
         INFO "Jepsen run DONE. END_TIME: $end_time"
-        PROCESS_RESULTS
```

Comment:
PROCESS_RESULTS removed is this correct?

Reply:
yes, results are processed with `./run.sh process-results`